README.md

```diff
@@ -1,5 +1,46 @@
 # TQ1 DOCS
 
-This repository contains all general tq1 docs that do not apply to the SDKs
+## What is TQ1?
 
-- master-reports-files : Read the docs [link](http://tq1-reports-files.readthedocs.org/)
+TQ1 a web platform made by Taqtile used to manage mobile-marketing campaigns. By using it you'll be able to create and schedule push notification campaigns, either by sending targeted notifications or by creating geofences that will target users according to location.
+
+It consists of 3 main parts: the SDKs, the admin and the server.
+
+In this docs you will find the links to all our documentation.
+
+To find more details on how to use TQ1 admin and FAQ, click [here](http://help.tq1.taqtile.io/) (you will have to login in order to view the content)
+
+## TQ1 basic flow
+
+The SDKs will send all information needed to the server, while the admin is used to create, manage and view results of  campaigns.
```

Comment:
This will be very confusing when you talk about Public API SDK, it might be better to use distinct names for `Mobile SDKs` and `Server SDKs` everywhere in the text